pkg/agent/agent.go

```diff
@@ -207,10 +208,14 @@ func (k *Klocksmith) process(stop <-chan struct{}) error {
 	glog.Infof("Deleting %d pods", len(pods))
 	deleteOptions := &v1meta.DeleteOptions{}
 	for _, pod := range pods {
-		glog.Infof("Terminating pod %q...", pod.Name)
-		if err := k.kc.CoreV1().Pods(pod.Namespace).Delete(pod.Name, deleteOptions); err != nil {
-			glog.Errorf("failed terminating pod %q: %v", pod.Name, err)
-			// Continue anyways, the reboot should terminate it
+		if strings.Contains(pod.Name, "update-agent") {
```

Comment:
hrm may want to do a label here  instead of a name match.  you can set a label that  says  drain-skip or something

Reply:
Why not do this in `k.getPodsForDeletion` ?